auth_api_key_group/models/auth_api_key_group.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Camptcamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+from odoo import fields, models
+
+
+class AuthApiKeyGroup(models.Model):
+    """Group API keys together."""
+
+    _name = "auth.api.key.group"
+    _description = "API Key auth group"
+
+    name = fields.Char(required=True)
+    code = fields.Char(required=True)
```

Comment:
I don't know the use case, but you'll probably search on `code`? No need for an index?

Reply:
not ATM